07.bowling_object/bowling_object.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+FRAME_COUNT = 10
+
+def main
+  # 入力
+  game = Game.new(ARGV[0])
+
+  p game.score
+end
+
+class Game
+  attr_reader :scores
+
+  def initialize(score)
+    @score = score
+    scores = @score.split(',')
+    # ショットをインスタンス化
+    @shots = scores.map.with_index(1) { |score, number| Shot.new(score, number) }
+
+    # frame毎に分割
+    frames = []
+    shots_copy = @shots.dup
+    (1..FRAME_COUNT).each do |f|
+     frame = []
+      if shots_copy[0].score == 10 && f < FRAME_COUNT
```

Comment:
IMO
個人的な好みですが、配列の最初を指定する際はindexではなくfirstなどのほうが好きです。
```suggestion
      if shots_copy.first.score == 10 && f < FRAME_COUNT
```